src/backend/bs_publish

```diff
@@ -658,7 +658,11 @@ sub createrepo_rpmmd {
     if (-e "$extrep/repodata/app-icons.tar") {
       print "    adding app-icons.tar to repodata\n";
       qsystem($modifyrepo_bin, "$extrep/repodata/app-icons.tar", "$extrep/repodata", @legacyargs) && die("    modifyrepo failed: $?\n");
-      unlink("$extrep/repodata/app-icons.tar");
+      qsystem("mv $extrep/repodata/app-icons.tar $extrep/repodata/appdata-icons.tar") && die("    copy appdata-icons failed: $?\n");
```

Comment:
You shouldn't call qsystem() with "mv xxx yyy" as one string, instead you should use 'mv', 'xxx', 'yyy'.
But in this specific case you shouldn't call qsystem() at all; just use perl's rename() function.

Reply:
Done. Sorry for the (huge) delay. Too much happening at the same time.